connexion/json_schema.py

```diff
@@ -128,6 +128,29 @@ def nullable_validation_fn(validator, to_validate, instance, schema):
     return nullable_validation_fn
 
 
+def validate_required(validator, required, instance, schema):
+    if not validator.is_type(instance, "object"):
+        return
+
+    for prop in required:
+        if prop not in instance:
+            properties = schema.get('properties')
+            if properties is not None:
+                subschema = properties.get(prop)
+                if subschema is not None:
+                    if 'readOnly' in validator.VALIDATORS and subschema.get('readOnly'):
+                        continue
+                    if 'writeOnly' in validator.VALIDATORS and subschema.get('writeOnly'):
```

Comment:
Please help me understand why this validation condition is detected and ignored: a field is required by the schema, the field is marked `writeOnly` in the schema, and the field is not present in the instance. I think in this situation, Connexion should declare error?